packages/ui/src/components/calendar/DateRangePresenter.svelte

```diff
@@ -318,7 +319,7 @@
     >
       {#if edits[2].value > -1}
         {edits[2].value.toString().padStart(4, '0')}
-      {:else}ГГГГ{/if}
+      {:else}YYYY{/if}
```

Comment:
Probably we need IntlString here

Reply:
Yes, https://front.hc.engineering/workbench/platform/tracker/TSK-465